CMakeLists.txt

```diff
@@ -137,6 +137,10 @@ if(CMAKE_C_COMPILER_ID MATCHES "Clang")
     if(HAVE_POISON_SYSTEM_DIRECTORIES_WARNING)
         add_definitions(-Wno-poison-system-directories)
     endif()
+    # MINGW defines printf with __attribute__ ((__unused__)). It not our fault so we just ignore it.
```

Comment:
Nit: change "defines" to "declares" and add "in stdio.h".

Change "It not our fault" to "It is out of our control".

Please add the URL of the MINGW issue you filed. Thanks!

Reply:
Updated. Again thanks for the advice!

Reply:
You're welcome! I found that GitHub UI allows me to edit the files directly. This is convenient because I won't need to trouble you for minor issues. I also got to edit the commit message when I did "Squash and commit" from the GitHub UI, although I still repeated the summary line by mistake :-)